FusionIIIT/templates/phcModule/appointment.html

```diff
@@ -1,63 +1,133 @@
+                
+                  <div class="field">
+                      <label>Doctor</label>
+                      <select id="doc" class="ui search fluid" name="doctor_id" required="true">
+                        <option selected hidden>--SELECT--</option>
+                        {% for doctor in doctors %}
+                        <option value="{{doctor.id}}">{{doctor.doctor_name}}</option>
+                        {% endfor %}
+                      </select>
+                   </div>
+                <div class="field">
+                  <label>Date of Appointment</label>
+                  <!-- on selection of doctor few days should be displayed -->
+                  
+                  
+                              <input type="date" placeholder="Select" id="date" name="date">
+                              <p id="dte"></p>
+                              <label>From Time</label>
+                              <input type="time" placeholder="Select" id="from_time" name="from_time">
+                              <p id="from_time"></p>
+                              <label>To Time</label>
+                              <input type="time" placeholder="Select" id="to_time" name="to_time">
+                              <p id="to_time"></p>
+
+
+                    
+                  
+              </div>
+
+
+                <div class="field">
+                    <label>Issues/Symptoms</label><p id="desc"></p>
+                    <div class="ui fluid input">
+                        <textarea id="description" class="ui textarea" rows="2" name="description" required="true"></textarea>
+                    </div>
+                </div>
+
+                <div class="field">
+                    <label><br></label>
+
+                    <div >
+                      <input type="submit" id="appo" name="Submit" value="Submit" class="ui small right floated primary button" />
+                    </div>
+
+                </div>
+            </form>
+          <script type="text/javascript" src="{% static 'globals/js/jquery.min.js' %}"></script>
+          <script type="text/javascript">
+
+
+
+             function trimfield(str)
+                                       {
+                                         return str.replace(/^\s+|\s+$/g,'');
+                                       }
+				 
+						
+					
+					
+					
+                $('#appo').on('click',function(e){
+                     e.preventDefault();
+
+                     var tod = new Date(new Date().setHours(0, 0, 0, 0));
+                     var today = new Date(new Date());
+                     var now = today.getHours();
+
+                     if(trimfield(document.getElementById("description").value) == '')
+                       {
+                       $('#desc').html('*Please fill out the Description!');
+                       document.getElementById("description").focus();
+                        }
+																										
+
```

Comment:
Please remove unnecessary whitespaces in the code.

Reply:
@akshatnema done. PTAL.